gencrud/config/object.py

```diff
@@ -40,28 +40,32 @@ def __init__( self,parent, **cfg ):
 
     @property
     def path( self ):
-        return self.__config.get( C_PATH, self.get_default( C_PATH ) )
+        return self.__config.get( C_PATH, self.get_default( 'path' ) )
 
     @property
     def cls( self ):
-        return self.__config.get( C_CLASS, self.get_default( C_MODULE ) )
```

Comment:
shouldn't it be `return self.__config.get( C_CLASS, self.get_default( C_CLASS ) )`

Reply:
keep module